dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -229,13 +233,19 @@ def runValidation(self, jobId, interfaces):
                             errorInterface.setRowErrorsPresent(jobId, True)
                         continue
                     valid, failures = Validator.validate(record,rules,csvSchema,fileType,interfaces)
-                    if(valid) :
+                    if valid:
                         try:
-                            tableObject.insert(record,fileType)
+                            record["job_id"] = jobId
+                            record["submission_id"] = submissionId
+                            # temporary fix b/c we can't use '+4' as a column alias :(
+                            if "primaryplaceofperformancezip+4" in record:
+                                record["primaryplaceofperformancezipplus4"] = record["primaryplaceofperformancezip+4"]
```

Comment:
I hope that this is temporary, do we expect the next version of the schema to not have the '+'?  Also, didn't we include replacing '+' with 'plus' in FieldCleaner?  That should make this unnecessary.